### PR TITLE
add acquire lock failure log for db mode and table columns name defined as constants

### PR DESCRIPTION
**Pull request by zjinlei**
### Ⅰ. Describe what this PR did
add log
table columns name defined as constants

### CODE REVIEW

File: core/src/main/java/io/seata/core/store/db/LockStoreDataBaseDAO.java [hunk 1]
```diff
@@ -119,8 +123,16 @@ public boolean acquireLock(List<LockDO> lockDOs) {
                 ps.setString(i + 1, lockDOs.get(i).getRowKey());
             }
             rs = ps.executeQuery();
+            String currentXID = lockDOs.get(0).getXid();
             while (rs.next()) {
-                if (!StringUtils.equals(rs.getString("xid"), lockDOs.get(0).getXid())) {
+                String dbXID = rs.getString("xid");
+                if (!StringUtils.equals(dbXID, currentXID)) {
+                    if (LOGGER.isInfoEnabled()) {
+                        String dbPk = rs.getString("pk");
```
Comment (slievrly) [inline core/src/main/java/io/seata/core/store/db/LockStoreDataBaseDAO.java]: Column names are best defined as constants

Comment (zjinlei) [reply core/src/main/java/io/seata/core/store/db/LockStoreDataBaseDAO.java]: done

File: rm-datasource/src/main/java/io/seata/rm/datasource/undo/UndoLogManager.java [hunk 2]
```diff
@@ -78,18 +79,24 @@ public int getValue() {
 
     private static String UNDO_LOG_TABLE_NAME = "undo_log";
 
+    /**
+     * branch_id, xid, context, rollback_info, log_status, log_created, log_modified
+     */
     private static String INSERT_UNDO_LOG_SQL = "INSERT INTO " + UNDO_LOG_TABLE_NAME +
```
Comment (slievrly) [inline rm-datasource/src/main/java/io/seata/rm/datasource/undo/UndoLogManager.java]: mark as final

File: rm-datasource/src/main/java/io/seata/rm/datasource/undo/UndoLogManager.java [hunk 3]
```diff
@@ -78,18 +79,24 @@ public int getValue() {
...
+            " VALUES (?, ?, ?, ?, ?, now(), now())";
 
     private static String DELETE_UNDO_LOG_SQL = "DELETE FROM " + UNDO_LOG_TABLE_NAME +
-        " WHERE branch_id = ? AND xid = ?";
+            " WHERE " + ClientTableColumnsName.UNDO_LOG_BRANCH_XID + " = ? AND " + ClientTableColumnsName.UNDO_LOG_XID + " = ?";
 
     private static String DELETE_UNDO_LOG_BY_CREATE_SQL = "DELETE FROM " + UNDO_LOG_TABLE_NAME +
         " WHERE log_created <= ? LIMIT ?";
 
     private static String SELECT_UNDO_LOG_SQL = "SELECT * FROM " + UNDO_LOG_TABLE_NAME +
```
Comment (slievrly) [inline rm-datasource/src/main/java/io/seata/rm/datasource/undo/UndoLogManager.java]: same as above

File: rm-datasource/src/main/java/io/seata/rm/datasource/undo/UndoLogManager.java [hunk 4]
```diff
@@ -384,18 +391,18 @@ public static String getCurrentSerializer() {
     }
 
     private static void insertUndoLogWithNormal(String xid, long branchID, String rollbackCtx,
-        byte[] undoLogContent, Connection conn) throws SQLException {
+                                                byte[] undoLogContent, Connection conn) throws SQLException {
```
Comment (slievrly) [inline rm-datasource/src/main/java/io/seata/rm/datasource/undo/UndoLogManager.java]: continuation indent=4

Comment (zjinlei) [reply rm-datasource/src/main/java/io/seata/rm/datasource/undo/UndoLogManager.java]: Configured no effect

File: rm-datasource/src/main/java/io/seata/rm/datasource/undo/UndoLogManager.java [hunk 5]
```diff
@@ -78,18 +79,24 @@ public int getValue() {
...
     private static String INSERT_UNDO_LOG_SQL = "INSERT INTO " + UNDO_LOG_TABLE_NAME +
-        " (branch_id, xid, context, rollback_info, log_status, log_created, log_modified)" +
-        " VALUES (?, ?, ?, ?, ?, now(), now())";
+            " (" + ClientTableColumnsName.UNDO_LOG_BRANCH_XID + ", " + ClientTableColumnsName.UNDO_LOG_XID + ", "
+            + ClientTableColumnsName.UNDO_LOG_CONTEXT + ", " + ClientTableColumnsName.UNDO_LOG_ROLLBACK_INFO + ", "
+            + ClientTableColumnsName.UNDO_LOG_LOG_STATUS + ", " + ClientTableColumnsName.UNDO_LOG_LOG_CREATED + ", "
+            + ClientTableColumnsName.UNDO_LOG_LOG_MODIFIED + ")" +
+            " VALUES (?, ?, ?, ?, ?, now(), now())";
 
     private static String DELETE_UNDO_LOG_SQL = "DELETE FROM " + UNDO_LOG_TABLE_NAME +
```
Comment (slievrly) [inline rm-datasource/src/main/java/io/seata/rm/datasource/undo/UndoLogManager.java]: mark as final

**Pull request merged**